documentation/docs/examples/image360.mdx

```diff
@@ -0,0 +1,70 @@
+
+```ts
+add360ImageSet(
+  datasource: 'events', 
+  eventFilter: { [key: string]: string }, 
+  add360ImageOptions?: AddImage360Options
+): Promise<Image360[]>;
+
+remove360Images(...image360Entities: Image360[]): Promise<void>:
+
+enter360Image(image360: Image360): Promise<void>:
+
+exit360Image(): void
+```
+
+To visualize 360 images the data must have been previously ingested into [Cognite Data Fusion](https://www.cognite.com/en/product/cognite_data_fusion_industrial_dataops_platform).
+More info on the ingestion process can be found [here](https://docs.cognite.com/cdf/3d/guides/360images_upload/).
+
+Once the data is ingested, a set of 360 images can be added to the viewer with `viewer.add360ImageSet(...)`.
+`datasource` describes the source data type used for storing the 360 images.
+At this time, the only valid `datasource` is `'events'`, but you can expect more to be added in the future.
+The `eventFilter` property describes the filter that is used when fetching data from the data source.
+This is a generic key-value pair and can point to any metdata that was set during ingestion.
+
+The `AddImage360Options` is used for correcting / adding transformations to the set of 360 images. The declaration can be found [here.](../api/modules/cognite_reveal#addimage360options)
+
+:::note
+The 360 images should be ingested with proper transformation data, and one should only rely on `AddImage360Options` when failing to do so or when using the same 360 images for multiple 3D models with different coordinate system.
+If you find that the data does not line up correctly, please try to update the source data instead adding the optional transformation.
+:::
+
+If the data is ingested similar to the guide referenced earlier, adding a set of 360 images can look like this:
+```ts
+viewer.add360ImageSet('events', { site_id: 'site-S01-area-a03' });
+```
+
+Once added to the viewer you will see a white circle representing each 360 image, here is an example of what it can look like:
+<img src={useBaseUrl('/img/image360_1.png')} alt="AxisViewTool" />
+
+Once the 360 images have been added, the icons are clickable, and when clicked, will enter the 360 image.
+Reveal provides navigation using mouse or touch and transition effects when clicking another 360 image.
+It is also possible enter a 360 image programatically by calling the `enter360Image(image360: Image360)` method.
+Exiting the visualization of a given 360 image is also possible using the `exit360Image()` method.
+For convenience, this is bound to the `ESCAPE` key out of the box.
+
+An example of entering into a 360 image can be seen here:
+<img src={useBaseUrl('/img/image360_2.png')} alt="AxisViewTool" />
+
+To remove one or more 360 images you can use the `remove360Images(...)` method which will also cleanup icons and other related data to the instance.
+
+The `Image360` declaration can be found here [here.](../api/interfaces/cognite_reveal.Image360)
+
```

Comment:
I realize that you start talking about the properties of the `Image360` interface as if they've already been mentioned :sweat_smile: It might be clearer if we add this: 

```suggestion
The `Image360` declaration can be found here [here.](../api/interfaces/cognite_reveal.Image360)

It includes a couple of `readonly` properties: `transform` of type `THREE.Matrix4` and `image360Visualization` of type `Image360Visualization`.
```